specification/web/resource-manager/Microsoft.Web/stable/2021-02-01/examples/GetWorkerApp.json

```diff
@@ -0,0 +1,61 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "name": "testWorkerApp0",
+    "api-version": "2021-02-01"
+  },
+  "responses": {
+    "200": {
+      "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.Web/workerApps/testWorkerApp0",
+      "name": "testWorkerApp0",
+      "type": "Microsoft.Web/workerApps",
+      "kind": "workerapp",
+      "location": "East US",
+      "properties": {
```

Comment:
>properties [](http://example.com/codeflow?start=7&length=10)

The schema definition includes `systemData` but it's not actually in the example response? Is that intentional?

Reply:
@majastrz - I cannot find systemData in the workerapps schema. Can you point me where it is?

Reply:
It's in the CommonDefinitions.json in the `Resource` definition if I remember right.